frontend/src/components/shared/CsgButton.vue

```diff
@@ -1,26 +1,32 @@
 <template>
   <button
-    :class="class"
+    :class="className"
     :type="btnType"
-    :disabled="loading"
+    :disabled="loading || disabled"
```

Comment:
should we add one more check here like:
`:disabled="loading || disabled || isDisabledClass"`